app/_hub/kong-inc/serverless-functions/0.3-x.md

```diff
@@ -0,0 +1,240 @@
+---
+name: Serverless Functions
+publisher: Kong Inc.
+version: 0.3-x
+
+source_url: https://github.com/Kong/kong-plugin-serverless-functions
+
+desc: Dynamically run Lua code from Kong during the access phase
+description: |
+  Dynamically run Lua code from Kong during access phase.
+
+type: plugin
+categories:
+  - serverless
+
+kong_version_compatibility:
+    community_edition:
+      compatible:
+        - 2.0.x
+        - 1.5.x      
+        - 1.4.x
+        - 1.3.x
+        - 1.2.x
+        - 1.1.x
+        - 1.0.x
+        - 0.14.x
+    enterprise_edition:
+      compatible:
+        - 1.5.x
+        - 1.3-x
+        - 0.36-x
+        - 0.35-x
+        - 0.34-x
+        - 0.33-x
+        - 0.32-x
+
+params:
+  name: pre-function OR post-function
+  service_id: true
+  route_id: true
+  consumer_id: false
+  protocols: ["http", "https"]
+  dbless_compatible: partially
+  dbless_explanation: |
+    The functions will be executed, but if the configured functions attempt to write to the database, the writes will fail.
+  config:
+    - name: functions
+      required: true
+      default: "[]"
+      value_in_examples: "[]"
+      description: Array of stringified Lua code to be cached and run in sequence during access phase.
+
+---
+
+## Plugin Names
+
+Serverless Functions come as two separate plugins. Each one runs with a
+different priority in the plugin chain.
+
+- `pre-function`
+  - Runs before other plugins run during access phase.
+- `post-function`
+  - Runs after other plugins in the access phase.
+
+## Demonstration
+
```

Comment:
Also, these sections could be enclosed in tabs to save on real estate.

{% navtabs %}
{% navtab With a database %}
{% endnavtab %}
{% navtab Without a database %}
{% endnavtab %}
{% endnavtabs %}

See https://github.com/Kong/docs.konghq.com/pull/2013/commits/d69449bdffdea03ed7b73748c0236048e4c7b9b6 for an example.

Reply:
done!